modes/play/objects/badGuy.js

```diff
@@ -0,0 +1,35 @@
+class BadGuy extends Mob {
+  constructor(game, mode, x, y) {
+    super(game, mode, x, y);
+    this.color = "#ee5";
+    this.brain = new BadGuyBrain(this);
+  }
+}
+
+
+class BadGuyBrain {
+  constructor(meat) {
+    this.meat = meat;
+    this.actionLoop = [
```

Comment:
These are just simple actions, that map to things that the meat actually knows how to do. In the future, these will likely be more complex functions, like "If I'm in range of a hostile creature, throw a Thing at it. If I'm not in range, find the best path to get in range of it."

Who knows what that will look like? I'm excited.